### PR TITLE
Updating opa gatekeeper documentation

### PR DESCRIPTION
**Pull request by parkand1**
Refresh of OPA Gatekeeper documentation:

1. Added usage section for adding gatekeeper to your blueprint 
2. Added section for key terminology
3. Used simpler example (all namespaces must be created with labels)
4. Added additional links for exploration

### CODE REVIEW

**Review by askulkarni2 — DISMISSED** [review]

File: docs/addons/opa-gatekeeper.md [hunk 1]
```diff
@@ -15,181 +15,87 @@ In the context of a Shared Services Platform running on Amazon EKS, platform tea
...
+```typescript
+import * as ssp from '@aws-quickstart/ssp-amazon-eks';
 
-input_containers[container] {
-  container := input.request.object.spec.containers[_]
-}
+const app = new cdk.App();
+const account = <AWS_ACCOUNT_ID>;
+const region = <AWS_REGION>;
+const env: { account, region },
```
Comment (shapirov103) [inline docs/addons/opa-gatekeeper.md:36]: you don't need env if it is only for account/region. You set those on lines 39 and 40.

**Review by shapirov103 — COMMENTED** [review]
Looks good, one tiny comment.

File: docs/addons/opa-gatekeeper.md (hunk 1 above)
Comment (parkand1) [reply docs/addons/opa-gatekeeper.md]: addressed

**Review by shapirov103 — APPROVED** [review]
lgtm. let's plan on a pattern that applies those policies with OPA (either through pipeline or through gitops).

**Pull request merged**